packages/mui-material/src/ToggleButtonGroup/toggleButtonGroupClasses.ts

```diff
@@ -16,6 +16,12 @@ export interface ToggleButtonGroupClasses {
   groupedVertical: string;
   /** Styles applied to the root element if `fullWidth={true}`. */
   fullWidth: string;
+  /** Styles applied to the first button in the button group. */
+  firstButton: string;
+  /** Styles applied to the last button in the button group. */
+  lastButton: string;
+  /** Styles applied to buttons in the middle of the button group. */
+  middleButton: string;
```

Comment:
To be more specific:
```suggestion
  /** Styles applied to the first button in the toggle button group. */
  firstButton: string;
  /** Styles applied to the last button in the toggle button group. */
  lastButton: string;
  /** Styles applied to buttons in the middle of the toggle button group. */
  middleButton: string;
```